lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -55,10 +58,66 @@ function LoggedInUser({ userInformation, setErrorState, zendeskApiHost, submitFo
       <option value="Other">{gettext('Other')}</option>
     </select>
   </div>);
+  if (showDiscussionButton) {
+    discussionElement = (
+      <div className="row">
+        <div className="col-sm-12">
+          <Button
+            className={['btn', 'btn-primary', 'btn-submit']}
+            onClick={reDirectUser}
+            label={gettext('Course Discussion Forum')}
+          />
+        </div>
+      </div>
+    );
+  }
+  if (showWarning) {
+    detailElement = (
+      <div id="warning-msg">
+        <div className="row">
+          <div className="col-sm-12">
+            <div className="form-group">
+              <p>
+                While our support team is happy to assist with the edX platform,
+                the course staff has the expertise for specific assignment questions
+                , grading or the proper procedures in each course.
+                Please post all course related questions within the Discussion
+                Forum where the Course Staff can directly respond.
+              </p>
```

Comment:
Shouldn't there be some i18n/translation based code here?

Reply:
Great question, there should be.